Makefile

```diff
@@ -47,8 +47,9 @@ down: ## Tears down the flask and harvester containers
 up-debug: ## Sets up local docker environment
 	docker compose -f docker-compose.yml -f docker-compose_debug.yml up -d
 
-clean: down ## Cleans docker images
+clean: ## Cleans docker images
```

Comment:
clean and down do similar things. could we just have clean?

Reply:
similar but not the same...

down takes the containers down
clean takes them down and deletes the data volumes associated with them.

Reply:
ah gotcha cool ok